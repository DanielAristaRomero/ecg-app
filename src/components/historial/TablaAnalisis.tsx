export default function TablaAnalisis() {
  return <div>TablaAnalisis</div>;
}
