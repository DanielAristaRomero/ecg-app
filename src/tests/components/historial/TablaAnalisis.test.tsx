import { shallow } from "enzyme";
import TablaAnalisis from "../../../components/historial/TablaAnalisis";

describe("Pruebas sobre TablaAnalisis.tsx", () => {
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {});
  test("debe rendirizar el componente correctamente", () => {
    const wrapper = shallow(<TablaAnalisis />);
    expect(wrapper).toMatchSnapshot();
  });
});
