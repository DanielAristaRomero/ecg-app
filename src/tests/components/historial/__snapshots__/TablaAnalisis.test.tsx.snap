// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre TablaAnalisis.tsx debe rendirizar el componente correctamente 1`] = `ShallowWrapper {}`;
